gradle/libs.versions.toml

```diff
@@ -0,0 +1,59 @@
+[versions]
+agp = "8.4.0"
```

Comment:
Minor: I would propose to keep each block sorted. Select from line 2 to line 16, and in Android Studio `Edit > Sort Lines`. Similarly for libraries and plugins